psi-ms.obo

```diff
@@ -22655,6 +22655,386 @@ relationship: has_metric_category MS:4000012 ! single run based metric
 relationship: has_value_type xsd:int ! The allowed value-type for this CV term
 relationship: has_units UO:0000189 ! count unit
 
+[Term]
+id: MS:4000108
+name: Peak density distribution MS1 Q1, Q2, Q3
+def: "From the distribution of peak densities in MS1, the quartiles Q1, Q2, Q3" [PSI:MS]
```

Comment:
**question:** Can you elaborate a bit about what the peak density values are? I don't immediately know how to interpret this description.

Reply:
Let's mark down some time in the next telco to refine the definition

Reply:
also deduplicated with quameter terms in existence already